### ISSUE TITLE
WIN10 BSOD(KERNEL_SECURITY_CHECK_FAILURE) on High IO 

### ISSUE THREAD

**Issue by gallexme** [maharmstone/btrfs#396]
for some reason it seems to work fine when using it regularly, but when i use for example robocopy with multiple threads to write onto an btrfs ssd or hdd, or use steam to download games at 1gbit it BSOD's with KERNEL_SECURITY_CHECK_FAILURE

3 minidumps with the BSOD

[minidumpszip.zip](https://github.com/maharmstone/btrfs/files/6720186/minidumpszip.zip)

**Comment by yretenai** [comment]
I'm blue screening about once a week for me, I suspect it's the driver and it's happening mostly because I have a lot of background processes running periodic tasks on the drives. Seems there may be some sort of race condition.

**Comment by yretenai** [comment]
#87 maybe related

**Comment by maharmstone** [comment]
Okay, reopening. Tibor, could you send me a minidump please?

**Comment by ScrewTSW** [comment]
> Okay, reopening. Tibor, could you send me a minidump please?

Absolutely. How do I do that? Is it auto-generated or is there a manual process to it?

**Comment by KOT4N** [comment]
> Tibor, could you send me a minidump please?

I can, same error - high io and btrfs.sys in module name

[021224-9390-01.zip](https://github.com/maharmstone/btrfs/files/14728166/021224-9390-01.zip)

**Comment by KOT4N** [comment]
Sry is old dump, but still btrfs
This is why I search for issues:
[032224-6890-01.zip](https://github.com/maharmstone/btrfs/files/14728214/032224-6890-01.zip)
